.github/workflows/push.yml

```diff
@@ -38,3 +38,26 @@ jobs:
             - uses: actions/checkout@v3
             - uses: ./.github/actions/setup-node
             - uses: ./.github/actions/build
+    deploy:
+        needs: build
+        name: Deploy
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v3
+            - uses: ./.github/actions/setup-node
+            - uses: ./.github/actions/deploy
+    release:
+        needs: deploy
+        name: Release
+        # Sets permissions of the GITHUB_TOKEN to allow opening release PR
+        permissions:
+            id-token: write
+            contents: write
+            packages: write
+            pull-requests: write
+            issues: read
```

Comment:
I believe these are need for the the changesets to work. I know at least pull-requests or packages is needed and Im not sure these will be used by the deploy action

Reply:
These are what people are using it seems https://github.com/changesets/action/issues/179#issuecomment-1139116421